server/session/player.go

```diff
@@ -87,9 +102,11 @@ const (
 	containerHotbar         = 27
 	containerInventory      = 28
 	containerOffHand        = 33
+	containerCraftingOffset = 46
```

Comment:
This is a list of container IDs, which the crafting offset, as far as I can tell, is not. Might want to call this something else (unless this is actually used as a container ID? In which case please do let me know).